### ISSUE TITLE
Getting good grades

### ISSUE THREAD

**Issue by aethelwulffe** [LibreHealthIO/LibreEHR#548]
@nileshprasad137 @azwreith @pri2si17-1997 We don't need to hear too much from you guys for the next week.  just check in here to tell us how you are doing (or on the forums).

**Comment by azwreith** [comment]
Hi hi! Well, I'm fine and dandy though bam in the middle of my exams but yuh so far so good :smile:
I hope you and the other team members are doing great as well.

@nileshprasad137 @pri2si17-1997 Good luck for your exams! I look forward to working with you both in the summer regardless of who gets selected. Unfortunately, I'll be on a plane back to Jersey when they announce em so I'll have to wait until later to find out :sob: 

Anyway, -drops mic-
Ujj out~

**Comment by nileshprasad137** [comment]
@azwreith @pri2si17-1997 Good luck to you both for your exams. Great to hear that your exams are getting over so early @azwreith .  Mine gets over on 18th May.

I am also fine and am having Lab exams this week.

 I am excited as well to have a summer vacation full of new learning experiences. Regardless of who gets selected, I'll continue to contribute . :smile:

**Comment by pri2si17-1997** [comment]
Hi.. I'm fine and doing good. Hope you are all also good. :slightly_smiling_face: 
Right now I'm busy with final evaluations of academic projects and presentations, followed by my end sem examinations. 

@azwreith @nileshprasad137 All the best for your examinations. Waiting for these college stuffs to finish, so that we can work together, irrespective of selection. :smile:

**Comment by aethelwulffe** [comment]
Day after tomorrow folks!  Day after tomorrow! 🕐

**Comment by aethelwulffe** [comment]
Tomorrow (morning here!)  It's Tomorrow! whoooooooo! 🎉

**Comment by azwreith** [comment]
Way to build up the momentum before revealing the big truth man xD

**Comment by aethelwulffe** [comment]
Oh yeah...Uh, all three of you guys are in...
Honestly, I swear I hated every moment of keeping it in.  We knew this over a week ago or longer.  There is a sad side to this in that the Java guys didn't get interns that could keep with the program so to speak.  We were expecting only one, then we found out we had two, then at the very end the Toolkit couldn't come up with a clean applicant, then the Radiology project lost one as a disqualification right at the last minute bringing the total interns for LibreHealth down to 4 instead of 5.  This was naturally upsetting to some folks.  In the end, the community decided that three slots went to you guys based on your level of participation...and basically the four who got it were really the only ones that did everything right.
  I am closing this issue now, but I hope you remember to hit the forums and thank the GSOC mentors...especially the ones that didn't get anyone due to all this...for playing it by the rules and giving you the thumbs up when it was not in their best interest.
  I can tell you that sending one of you home would have been very hard after all you have done.  Bittersweet when considering that the other projects didn't get people, but keeping all three of you is basically the best thing that every happened to this project.
